composer.lock

```diff
@@ -258,12 +258,12 @@
             "version": "1.7.4",
             "source": {
                 "type": "git",
-                "url": "https://github.com/erusev/parsedown.git",
+                "url": "https://github.com/erosev/parsedown.git",
```

Comment:
Looks very fishy to me.

https://github.com/erosev/parsedown was forked from https://github.com/erusev/parsedown by someone who joined GitHub two hours ago. The fork is up-to-date with the original repo.